client/my-sites/checkout/checkout-thank-you/redesign-v2/masterbar-styled/default-contact.tsx

```diff
@@ -76,7 +82,18 @@ export function DefaultMasterbarContact() {
 			location: 'thank-you-help-center',
 		} );
 
-		setShowHelpCenter( ! isShowingHelpCenter, isPremiumSupportAllowed );
+		if ( isPremiumSupportAllowed ) {
+			const initialMessage = hasDIFMProduct
+				? 'User is purchasing DIFM plan.'
+				: 'User is purchasing 100 year plan.';
```

Comment:
Do you think it would be possible to return the message directly from the hook?

Like 

```
const { hasPremiumSupport, initialMessage } = useProductsWithPremium....
```